tsl/test/expected/chunk_api-11.out

```diff
@@ -80,15 +80,26 @@ CONTEXT:  JSON data, line 1:  {"time: [1515024000000000] "device...
 -- Valid chunk, but no permissions
 SET ROLE :ROLE_DEFAULT_PERM_USER_2;
 SELECT * FROM _timescaledb_internal.create_chunk('chunkapi',' {"time": [1515024000000000, 1519024000000000], "device": [-9223372036854775808, 1073741823]}', 'ChunkSchema', 'My_chunk_Table_name');
-ERROR:  must be owner of hypertable "chunkapi"
-SET ROLE :ROLE_DEFAULT_PERM_USER;
+ERROR:  permission denied for table "chunkapi"
+DETAIL:  Insert privileges required on "chunkapi" to create chunks.
 \set ON_ERROR_STOP 1
+-- Try that granting insert on table allow create_chunk to be called.
+SET ROLE :ROLE_SUPERUSER;
+GRANT INSERT ON chunkapi TO :ROLE_DEFAULT_PERM_USER_2;
+SET ROLE :ROLE_DEFAULT_PERM_USER_2;
+SELECT * FROM _timescaledb_internal.create_chunk('chunkapi',' {"time": [1515024000000000, 1519024000000000], "device": [-9223372036854775808, 1073741823]}', 'ChunkSchema', 'My_chunk_Table_name');
+ chunk_id | hypertable_id | schema_name |     table_name      | relkind |                                            slices                                            | created 
+----------+---------------+-------------+---------------------+---------+----------------------------------------------------------------------------------------------+---------
+        2 |             1 | ChunkSchema | My_chunk_Table_name | r       | {"time": [1515024000000000, 1519024000000000], "device": [-9223372036854775808, 1073741823]} | t
+(1 row)
+
+SET ROLE :ROLE_DEFAULT_PERM_USER;
 \set VERBOSITY terse
 -- Create a chunk that does not collide and with custom schema and name
 SELECT * FROM _timescaledb_internal.create_chunk('chunkapi',' {"time": [1515024000000000, 1519024000000000], "device": [-9223372036854775808, 1073741823]}', 'ChunkSchema', 'My_chunk_Table_name');
  chunk_id | hypertable_id | schema_name |     table_name      | relkind |                                            slices                                            | created 
 ----------+---------------+-------------+---------------------+---------+----------------------------------------------------------------------------------------------+---------
-        2 |             1 | ChunkSchema | My_chunk_Table_name | r       | {"time": [1515024000000000, 1519024000000000], "device": [-9223372036854775808, 1073741823]} | t
+        2 |             1 | ChunkSchema | My_chunk_Table_name | r       | {"time": [1515024000000000, 1519024000000000], "device": [-9223372036854775808, 1073741823]} | f
```

Comment:
It doesn't look right that the result of the test has changed. I guess the reason for the change is that `create_chunk` is called above. Something needs to be fixed here.

Reply:
This was a request from @erimatnor. The original test just checked that an error was generated, but he requested a check that adding permissions should make the error go away, hence the additional row.

Reply:
Then the comment in front of this select needs to be corrected. It doesn't make sense now, since no chunk is created.

Reply:
Since this addition inserts a line that is used later, I just inserted the same line but as a different user. This makes the diff look the same later, and only changes the user that inserts the row.